README.md

```diff
@@ -6,11 +6,8 @@ In this workshop we'll be building on what we learnt in the [pg walkthrough](htt
 
 - Clone this repo
 - Install the node_modules by typing `npm install` in your terminal.
-- Run `npm run dev`in your terminal and checkout the result at [http://localhost:5000]. This is a preview of what you'll be building.
-- Create a database locally
-- Add a users table.
-- Create a table called users with 3 columns.
-- Install the npm module `pg` , `npm install --save pg`.
+- Run `npm run dev`in your terminal and checkout the result at [http://localhost:5000]. This is the starting template for what you'll be building.
+- Install the npm modules `pg` and `env2`: `npm install --save pg env2`.
```

Comment:
cant these just be in the package they are pulling down? At that point `npm install` will grab them..?